crates/csharp/src/lib.rs

```diff
@@ -3292,28 +3333,111 @@ fn is_primitive(ty: &Type) -> bool {
 }
 
 trait ToCSharpIdent: ToOwned {
+    fn csharp_keywords() -> Vec<&'static str>;
     fn to_csharp_ident(&self) -> Self::Owned;
+    fn to_csharp_ident_upper(&self) -> Self::Owned;
 }
 
 impl ToCSharpIdent for str {
+    // Source: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/
+    // TODO: Repace with actual keywords
```

Comment:
This todo should be have been resolved a long time ago I think?

Reply:
Correct, I have now removed this comment.